### ISSUE TITLE
Organise the code into module

### ISSUE THREAD

**Issue by glemaitre** [pengsongyou/Seeded_Segmentation#2]
Hi again,

After checking, the first striking thing is that you have only one main file.
You need to organise the code into submodule dedicated to specific task.
It will be more readable.

Cheers,

**Comment by pengsongyou** [comment]
Thank you very much! Now the code is only implementation of algorithm, and we will definitely improve it, adding GUI, reorganise the code. Thank you!

**Comment by glemaitre** [comment]
Yes do not worry. I am just doing a round of commenting. In case that it could be useful.
You still have time :)
